src/components/signup-form.js

```diff
@@ -78,6 +78,19 @@ var Signup = React.createClass({
     return this.stickyContainer.getClientRects()[0].top + this.stickyContent.offsetHeight + window.scrollY - window.innerHeight;
   },
   render: function() {
+    var siteCountries = ['AT', 'BE', 'BG', 'HR', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'HU', 'IE', 'IT', 'LV', 'LT', 'LU', 'MT', 'NL', 'PL', 'PT', 'RO', 'SK', 'SI', 'ES', 'SE', 'GB'];
+    var supportedLocales = require('../../node_modules/localized-countries/languages.json');
+    var selectedLocale = 'en';
+    var targetLocale = this.context.intl.locale.replace(/-/g , "_");
+    if (supportedLocales.indexOf(targetLocale) >= 0) {
+      selectedLocale = targetLocale;
+    }
+    var localizedCountriesAll = require('../../node_modules/localized-countries/data/' + selectedLocale + '.json');
```

Comment:
How is this working? I thought we couldn't do require calls with dynamic values?